misc-scripts/xref_mapping/XrefParser/Mim2GeneParser.pm

```diff
@@ -1,8 +1,7 @@
 =head1 LICENSE
 
-Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
-Copyright [2016-2019] EMBL-European Bioinformatics Institute
-
+See the NOTICE file distributed with this work for additional information
```

Comment:
Do we have a notice file?
Either needs changing here or in my porting PRs...

Reply:
We do now! I've just pushed it straight to _feature/xref_sprint_ - it's an administrative thing and the content of that file have already been agreed upon, back when it got added to _ensembl-xref_.

Reply:
great! will update my backport PRs